lib/ansible/cli/__init__.py

```diff
@@ -294,9 +296,8 @@ def base_parser(usage="", output_opts=False, runas_opts=False, meta_opts=False,
             help="verbose mode (-vvv for more, -vvvv to enable connection debugging)")
 
         if inventory_opts:
-            parser.add_option('-i', '--inventory-file', dest='inventory',
-                help="specify inventory host path (default=%s) or comma separated host list." % C.DEFAULT_HOST_LIST,
-                default=C.DEFAULT_HOST_LIST, action="callback", callback=CLI.expand_tilde, type=str)
+            parser.add_option('-i', '--inventory-file', dest='inventory', action="append",
```

Comment:
it never was, since 'host list' was always accepted. I'll add it and make note of deprecating `--inventory-fie`